.env

```
@@ -0,0 +1 @@
+NEXT_PUBLIC_API="http://localhost:5080/api"
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)